scapy/utils.py

```diff
@@ -816,6 +817,8 @@ def next(self):
         return pkt
     __next__ = next
 
+    PacketMetadata = collections.namedtuple("PacketInfo",
```

Comment:
The types (both this one and `BlockMetadata`) should not be related to the object (sorry I had missed that). Could you declare them outside the class declaration? Also, could you set their name to "PacketMetadata" and "BlockMetadata"?

Reply:
Already fixed.

Reply:
..but what about RawPcapNgReader.BlockMetadata name which I used?
Maybe better if will be the same (PacketMetadata) as in RawPcapReader ?

Reply:
:: The types (both this one and BlockMetadata) should not be related to the object (sorry I had missed that).

Cannot agree. Its are used only in this class (as a result only this class methods), so it is specific for this class/type only.
This idiom is very often (especially because Python has no namespaces) and quite correct.
Using such names in the module namespace/scope has less sense.
Much better is:
RawPcapReader.BlockMetadata than RawPcapReader_BlockMetadata and
RawPcapNgReader.BlockMetadata than RawPcapNgReader_BlockMetadata